docs/src/components/home/Sponsors.tsx

```diff
@@ -8,46 +8,81 @@ import Paper from '@material-ui/core/Paper';
 import Link from 'docs/src/modules/components/Link';
 import IconButton from '@material-ui/core/IconButton';
 import AddRounded from '@material-ui/icons/AddRounded';
+import LaunchRounded from '@material-ui/icons/LaunchRounded';
 
 const DIAMONDs = [
   {
-    logo: 'https://avatars3.githubusercontent.com/u/1287123?s=256',
+    src: 'https://avatars3.githubusercontent.com/u/1287123?s=40',
+    srcSet: 'https://avatars3.githubusercontent.com/u/1287123?s=80 2x',
     name: 'Octopus Deploy',
     description: 'Repetable relayable deployments.',
     href: 'https://octopus.com/',
   },
   {
-    logo: 'https://avatars3.githubusercontent.com/u/8424863?s=256',
+    src: 'https://avatars3.githubusercontent.com/u/8424863?s=40',
+    srcSet: 'https://avatars3.githubusercontent.com/u/8424863?s=80 2x',
     name: 'Doit International',
     description: 'Management platform for Google Clound and AWS.',
     href: 'https://www.doit-intl.com/',
   },
 ];
 const GOLDs = [
   {
-    logo: 'https://github.com/tidelift.png?size=96',
+    src: 'https://avatars.githubusercontent.com/u/30204434?s=40',
+    srcSet: 'https://avatars.githubusercontent.com/u/30204434?s=80 2x',
     name: 'Tidelift',
     description: 'Enterprise-ready open source software.',
     href: 'https://tidelift.com/',
   },
   {
-    logo: 'https://github.com/teambit.png?size=96',
+    src: 'https://avatars.githubusercontent.com/u/24789812?size=40',
+    srcSet: 'https://avatars.githubusercontent.com/u/24789812?size=80 2x',
     name: 'Bit',
     description: 'The fastest way to share code.',
     href: 'https://bit.dev/',
   },
   {
-    logo: 'https://images.opencollective.com/callemall/a6946da/logo/96.png',
+    src: 'https://images.opencollective.com/callemall/a6946da/logo/40.png',
+    srcSet: 'https://images.opencollective.com/callemall/a6946da/logo/80.png 2x',
     name: 'Text-em-all',
     description: 'The easy way to message your group.',
     href: 'https://www.text-em-all.com/',
   },
   {
-    logo: 'https://images.opencollective.com/canadacasino/5b19004/logo/96.png',
+    src: 'https://images.opencollective.com/canadacasino/5b19004/logo/40.png',
+    srcSet: 'https://images.opencollective.com/canadacasino/5b19004/logo/80.png 2x',
     name: 'Canada Casino',
     description: 'Safe and rewarding online casino experience',
     href: 'https://casinocanada.com/',
   },
+  {
+    src: '/static/sponsors/spice-factory.png',
+    srcSet: '/static/sponsors/spice-factory-2x.png',
+    name: 'Spice Factory',
+    description: 'Next gen digital product studio.',
+    href: 'https://spicefactory.co/',
+  },
+  {
+    src: '/static/sponsors/elevator-logo.png',
+    srcSet: '/static/sponsors/elevator-logo-2x.png',
+    name: 'Elevator',
+    description: 'The dopest new hip hop, upcoming artsits, music.',
+    href: 'https://www.elevatormag.com/',
+  },
+  {
+    src: '/static/sponsors/movavi.png',
+    srcSet: '/static/sponsors/movavi-2x.png',
```

Comment:
```suggestion
    src: 'https://images.opencollective.com/movavi-software/a1d0167/logo/40.png',
    srcSet: 'https://images.opencollective.com/movavi-software/a1d0167/logo/80.png 2x',
```

Reply:
the image is different, won't change for now.

Reply:
They are different because movavi decided it to. They want us to sponsor the iPhone app taking, not their company, for some reason.

Reply:
got it.